Ginger/Ginger/MainWindow.xaml.cs

```diff
@@ -381,6 +381,13 @@ private void AppCleanUp()
             CW.Show();
             GingerCore.General.DoEvents();
 
+            if(WorkSpace.Instance!=null && WorkSpace.Instance.Solution!=null && WorkSpace.Instance.Solution.LoggerConfigurations!=null)
+            {
+                while (WorkSpace.Instance.Solution.LoggerConfigurations.IsPublishToCentralDBRunning == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)